### PR TITLE
Update rubocop to 0.6.x.

### PR DESCRIPTION
**Pull request by djberg96**
Gemfile update that bumps us up to rubocop 0.6.x. This will require some simultaneous updates to the guide:

~~https://github.com/ManageIQ/guides/pull/343~~
https://github.com/ManageIQ/guides/pull/351

### CODE REVIEW

**Comment by martinpovolny** [comment]
https://github.com/ManageIQ/guides/pull/351

**Comment by Fryguy** [comment]
@bdunne Please review.

File: Gemfile [hunk 1]
```diff
@@ -49,7 +49,8 @@ gem 'awesome_spawn',        '>= 1.4.1'
 gem 'default_value_for'
 gem 'haml_lint',            '~> 0.20.0', :require => false
 gem 'more_core_extensions', '~> 2.0.0',  :require => 'more_core_extensions/all'
-gem 'rubocop',              '~> 0.52.0', :require => false
+gem 'rubocop',              '~> 0.69.0', :require => false
+gem 'rubocop-performance',  '~> 1.3',    :require => false
```
Comment (Fryguy) [inline Gemfile]: What is this gem?

File: Gemfile.lock [hunk 2]
```diff
@@ -178,11 +179,10 @@ GEM
       mini_portile2 (~> 2.3.0)
     octokit (4.8.0)
       sawyer (~> 0.8.0, >= 0.5.3)
-    parallel (1.12.1)
-    parser (2.5.0.2)
+    parallel (1.17.0)
```
Comment (Fryguy) [inline Gemfile.lock]: Why did parallel update? (I'm ok with it, just curious what caused it to update.

File: Gemfile (hunk 1 above)
Comment (djberg96) [reply Gemfile]: Courtesy of Martin P:

```
Post-install message from rubocop:
Performance cops have been removed from RuboCop 0.68. Use the rubocop-performance gem instead.

Put this in your Gemfile.

gem 'rubocop-performance'
And then execute:

$ bundle install
Put this into your .rubocop.yml.

require: rubocop-performance
More information: https://github.com/rubocop-hq/rubocop-perfor
```

File: Gemfile.lock (hunk 2 above)
Comment (djberg96) [reply Gemfile.lock]: I just did `bundle update rubocop` and that's what it gave me. *shrug*

File: Gemfile (hunk 1 above)
Comment (djberg96) [reply Gemfile]: I'll see if I can get Martin to update the rubocop file on his end as well.

Comment (Fryguy) [reply Gemfile]: Ah thanks!  I noticed that rspec org also has split out Rails and Rspec gems as well...seems they are going towards the pluggable approach.

File: Gemfile.lock (hunk 2 above)
Comment (Fryguy) [reply Gemfile.lock]: I'm sure it's fine :+1:

**Pull request merged**